feathr_project/feathr/client.py

```diff
@@ -5,6 +5,8 @@
 from datetime import datetime, timedelta
 from pathlib import Path
 from typing import Dict, List, Optional, Union
+from pprint import pprint
+from xmlrpc.client import Boolean
```

Comment:
this seems wrong wrong?

Reply:
Oops sorry. Removed `from xmlrpc.client import Boolean`